Platform.PCL/Realm.PCL/RealmPCL.cs

```diff
@@ -292,14 +292,17 @@ public void Write(Action action)
             RealmPCLHelpers.ThrowProxyShouldNeverBeUsed();
         }
 
+
         /// <summary>
-        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread. If no exception is thrown,
+        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread, <b>if</b> called from UI thread. If no exception is thrown,
```

Comment:
While that would be the majority of the cases, it's realistic to have `SynchronizationContext` on non-UI threads, so the updated text might be a little misleading.